frame/contracts/src/weights.rs

```diff
@@ -209,262 +201,264 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: System Account (r:1 w:1)
 	// Storage: Contracts OwnerInfoOf (r:1 w:1)
 	fn instantiate(s: u32, ) -> Weight {
-		(236_193_000 as Weight)
+		(224_708_000 as Weight)
 			// Standard Error: 2_000
-			.saturating_add((2_022_000 as Weight).saturating_mul(s as Weight))
+			.saturating_add((2_028_000 as Weight).saturating_mul(s as Weight))
 			.saturating_add(T::DbWeight::get().reads(6 as Weight))
 			.saturating_add(T::DbWeight::get().writes(5 as Weight))
 	}
 	// Storage: Contracts ContractInfoOf (r:1 w:1)
 	// Storage: Contracts CodeStorage (r:1 w:0)
 	// Storage: Timestamp Now (r:1 w:0)
 	// Storage: System Account (r:1 w:1)
-	fn call() -> Weight {
-		(189_193_000 as Weight)
+	fn call(c: u32, ) -> Weight {
+		(388_105_000 as Weight)
```

Comment:
Oh this didn't work out how I hoped. I expected that this weight should stay the same because the weight shouldl be the same as before for `c = 0`.